runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java

```diff
@@ -125,6 +128,20 @@ public void onWidgetLoaded(WidgetLoaded event)
 
 				if (config.waveTimes() && rewardWidget != null && rewardWidget.getText().contains(ENDGAME_REWARD_NEEDLE_TEXT) && gameTime != null)
 				{
+					try
+					{
+						double currentpb = configManager.getRSProfileConfiguration("personalbest", "barbarian assault", double.class);
+						double newpb = gameTime.getPBTime();
+						if (newpb < currentpb)
+						{
+							configManager.setRSProfileConfiguration("personalbest", "barbarian assault", gameTime.getPBTime());
+						}
+					}
+					catch (Exception e)
```

Comment:
don't use exceptions for normal control flow

Reply:
Took the exception out of normal control flow